tests/acceptance/bootstrap/NotificationContext.php

```diff
@@ -582,14 +585,16 @@ public function userShouldHaveReceivedTheFollowingEmailFromUserIgnoringWhitespac
 	 * @param string $user
 	 * @param string $expectedEmailBodyContent
 	 * @param bool $ignoreWhiteSpace
+	 * @param bool $contains
 	 *
 	 * @return void
 	 * @throws GuzzleException
 	 */
 	public function assertEmailContains(
 		string $user,
 		string $expectedEmailBodyContent,
-		$ignoreWhiteSpace = false
+		bool $ignoreWhiteSpace = false,
+		bool $contains = true,
```

Comment:
```suggestion
		bool $emailShouldContain = true,
```
i am not aware if this boolean is either email should be receive or email body should contain